### PR TITLE
T-1 prepare repo 

### LINKED ISSUE #1: Prepare repo for development
Create project files for local IDEs. 
Last meeting we decided to use STM32CubeIDE for HW binary creation and debug. 
Python

### PR DESCRIPTION
**Pull request by IBeRyUS**
Prepare repo with emulator(Python) and client(C) connection modules.

### CODE REVIEW

File: Core/Inc/main.h [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+#define USART_RX_GPIO_Port GPIOA
+#define LD2_Pin GPIO_PIN_5
+#define LD2_GPIO_Port GPIOA
+#define TMS_Pin GPIO_PIN_13
+#define TMS_GPIO_Port GPIOA
+#define TCK_Pin GPIO_PIN_14
+#define TCK_GPIO_Port GPIOA
+#define SWO_Pin GPIO_PIN_3
+#define SWO_GPIO_Port GPIOB
+/* USER CODE BEGIN Private defines */
```
Comment (zaya-mc) [inline Core/Inc/main.h]: I would suppose it is just an example project and you don't care about portability.

**Review by zaya-mc — COMMENTED** [review]
It is difficult to review. You might push the ST HAL libraries to the remote master without any PR, then you might create a PR for what you implemented?

File: Core/Inc/main.h (hunk 1 above)
Comment (IBeRyUS) [reply Core/Inc/main.h]: This is auto generated code from CubeMX. Only generated to test initial hw. Driver section might take care of portability. The main portability layer should be implemented for layer above drivers

**Comment by IBeRyUS** [comment]
Updated the PR and removed SDK files.
And finally added first CI :)

**Review by zaya-mc — APPROVED** [review]

**Pull request merged**